examples/gateway/anthropic/README.md

```diff
@@ -0,0 +1,27 @@
+## Example route configuration for Anthropic
+
+To set up your MLflow configuration file, include a single route for the completions endpoint as follows:
+
+```yaml
+routes:
+  - name: completions-claude
```

Comment:
```suggestion
  - name: completions
```

Reply:
thanks for the catch :) that was an artifact of manual testing with my gateway server